src/main/java/randoop/generation/ForwardGenerator.java

```diff
@@ -93,6 +96,13 @@ public ForwardGenerator(
     this.instantiator = componentManager.getTypeInstantiator();
 
     initializeRuntimePrimitivesSeen();
+
+    if (GenInputsAbstract.enable_bloodhound) {
+      // If Bloodhound is enabled, select the next operation while considering the methods' weights.
+      this.operationSelector = Bloodhound.instance;
```

Comment:
Shouldn't this create the Bloodhound instance, just as it creates the UniformRandomMethodSelection instance two lines later?